test/k8sT/DatapathConfiguration.go

```diff
@@ -319,6 +319,48 @@ var _ = Describe("K8sDatapathConfig", func() {
 		})
 	})
 
+	Context("IPVLAN", func() {
+		BeforeEach(func() {
+			SkipIfIntegration(helpers.CIIntegrationEKS)
+			SkipIfIntegration(helpers.CIIntegrationFlannel)
+			SkipIfIntegration(helpers.CIIntegrationGKE)
+		})
+
+		It("Check connectivity with IPVLAN", func() {
+			options := map[string]string{
+				"tunnel":              "disabled",
+				"datapathMode":        "ipvlan",
+				"ipvlan.masterDevice": "eth0",
```

Comment:
I believe we don't have an `eth0` device in our test VMs. Will this still work?

Reply:
You could use the same logic for detecting devices as we have `overwriteHelmOptions()` function.